Tests/SentryTests/Protocol/SentrySDKSettings+Equality.h

```diff
@@ -0,0 +1,13 @@
+#import "SentrySDKSettings.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentrySDKSettings (Equality)
```

Comment:
`l`: Is there a reason why this is not part of the SDK?

Reply:
This is only needed for testing.